### PR TITLE
Tutorial Update

### PR DESCRIPTION
**Pull request by ssktotoro**
* Verified Training for Mindboggle, GMWM, and Atlas Segmentation
* Published minimal_example in a notebook as well as a python file
* Added pretrained models
* Added example segmentation notebooks
* Removed extra files
* Added dropout and large MeshNet model configurability

### CODE REVIEW

**Review by ssktotoro — COMMENTED** [review]
LGTM
